### DISCUSSION TITLE
Dropdown always visible

### DISCUSSION THREAD

**Discussion by EdouardPan** [orchidjs/tom-select#387] [Q&A] [answered]
Hello everyone,

Is there a way to have a dropdown always visible under the select input in the page. What I mean is:
- You do not have to click on the input to display the dropdown and if you click somewhere else it stays visible
- The dropdown has its own space, it does not overlap/hide other elements below

I hope I am enough clear! Do not hesitate to tell me otherwise, I'll draw something.

Thank you!

**Comment by oyejorge** [comment, marked as answer]
See the Inline Display example on https://tom-select.js.org/examples/remote/

**Reply by EdouardPan** [reply]
Perfect! Thanks!
